apps/core/templates/components/footer.html

```diff
@@ -38,14 +39,14 @@ <h2 class="link-block__heading">Help improve this guide</h2>
                     <div class="link-block__content">
                         <h2 class="link-block__heading">Wagtail</h2>
                         <div class="link-block__description">
-                            Visit Wagtail.org for more resources and Wagtail news
+                            {% trans 'Visit Wagtail.org for more resources and Wagtail news' %}
```

Comment:
Here and above – I’d have expected all of this to be done with a Footer snippet.

Reply:
This should be covered in #177, I'll rebase this PR once that's merged.